docs/layout-of-source-files.rst

```diff
@@ -208,5 +208,6 @@ for the two input parameters and two returned values.
         function rectangle(uint w, uint h) returns (uint s, uint p) {
             s = w * h;
             p = 2 * (w + h);
+            return (s, p);
```

Comment:
The example is actually working, assigning to the return variables will result in returning those.